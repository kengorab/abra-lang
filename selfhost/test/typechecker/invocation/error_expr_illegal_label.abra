func foo(fn: (Int) => Int): Int = fn(hmm: 123)
