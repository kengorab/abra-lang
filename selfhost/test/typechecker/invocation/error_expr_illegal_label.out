Error at %FILE_NAME%:1:38
Incorrect use of label for parameter
  |  func foo(fn: (Int) => Int): Int = fn(hmm: 123)
                                          ^
Parameter labels are not allowed here because parameters' names cannot be determined
